test/app.test.js

```diff
@@ -67,6 +83,37 @@ describe('/account/sendRecoveryMessage', () => {
         expect(text).toMatch(new RegExp(`https://wallet.nearprotocol.com/recover-with-link/${ctx.accountId}/${SEED_PHRASE.replace(/ /g, '%20')}`));
     });
 
+    test('setting multiple recovery methods', async () => {
+        const email = 'test@dispostable.com';
+        await request.post('/account/sendRecoveryMessage')
+            .send({
+                accountId: ctx.accountId,
+                email,
+                seedPhrase: SEED_PHRASE
+            });
+        let account = await models.Account.findOne({
+            where: { accountId: ctx.accountId }
+        });
+        await account.reload();
+        expect(account.email).toBe(email);
+        expect(account.emailAddedAt).toBeTruthy();
+
+
+        const phoneNumber = '+1.800.867.5309';
+        await request.post('/account/sendRecoveryMessage')
+            .send({
+                accountId: ctx.accountId,
+                phoneNumber,
+                seedPhrase: SEED_PHRASE
+            });
+        account = await models.Account.findOne({
+            where: { accountId: ctx.accountId }
+        });
+        await account.reload();
+        expect(account.phoneNumber).toBe(phoneNumber);
```

Comment:
This test fails intermittently on this line and I'm not sure why – my best guess is a caching issue with `sequelize`. Note that I've already added `reload()` calls _in addition_ to the `findOne` lookups – I would think that both are not necessary, but in my testing I found that data would be consistently stale without both `reload()` calls. 😒 

I'm not sure how to make this test more robust than it already is, or how to restructure it to adequately test this functionality without running into caching issues. Ideas?

Reply:
Likely caused by missing `await`, `.reload()` likely helps just by accident because it creates more delay.